.cloudtest.yaml

```diff
@@ -22,6 +22,7 @@ retest:  # Allow to do test re-run if some kind of failures are detected, line C
     - "NetworkPlugin cni failed to set up pod"    # Error in AWS dur to leak of IPs or not ability to assign them.
     - "etcdserver: request timed out, possibly due to previous leader fail"  # Error in any could, reason unknown.
     - "unable to establish connection to VPP (VPP API socket file /run/vpp/api.sock does not exist)"  # a VPP is not started, it will be re-started in general, but will cause test fail.
+    - "node(s) didn't match node selector"
```

Comment:
Is this issue related to CNI AWS problem or it means that we did not free some resources during testing?

Reply:
Based on what I found it could be related to device plugin<->Kubernetes communication issue.